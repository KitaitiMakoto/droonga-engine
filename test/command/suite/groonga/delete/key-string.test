#@include fixture/string-key-table.jsons
{
  "type": "add",
  "dataset": "Default",
  "body": {
    "table"  : "String",
    "key"    : "key"
  }
}
{
  "type": "delete",
  "dataset": "Default",
  "body": {
    "table" : "String",
    "key"   : "key"
  }
}
