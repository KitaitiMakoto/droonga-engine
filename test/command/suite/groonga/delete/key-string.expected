{
  "inReplyTo": "request-id",
  "statusCode": 200,
  "type": "add.result",
  "body": true
}
{
  "inReplyTo": "request-id",
  "statusCode": 200,
  "type": "delete.result",
  "body": [
    [
      0,
      0.0,
      0.0
    ],
    true
  ]
}
